NAMESPACE

```diff
@@ -2,7 +2,20 @@
 
 export("%>%")
 export(add_external_resources)
+export(aggregate_data)
 export(app_server)
 export(app_ui)
+export(compute_indicator)
+export(compute_national_indicator)
+export(filter_data)
+export(ki_table)
+export(load_data)
+export(mod_kvalitetsoversikt_server)
+export(mod_kvalitetsoversikt_ui)
 export(run_app)
+export(table_row_constructor)
 importFrom(magrittr,"%>%")
+importFrom(rlang,.data)
+importFrom(shiny,NS)
```

Comment:
Is `importFrom(shiny,NS)` still needed?

Reply:
NAMESPACE is generated from function docs and any changes should be made there...